maintenance_mode/views.py

```diff
@@ -13,7 +13,9 @@ def maintenance_mode_off(request):
     if request.user.is_superuser:
         set_maintenance_mode(False)
 
-    return HttpResponseRedirect(request.META['SCRIPT_NAME'] or '/')
+    redirect_to = request.META.get('SCRIPT_NAME', '/')
+
```

Comment:
It would be better to update the variable outside the response constructor considering that the trailing slash should be appended only if `settings.APPEND_SLASH = True`.